### ISSUE TITLE
Accessibility: "Add content" buttons not accessible via keyboard

### ISSUE THREAD

**Issue by glombek** [PerplexDigital/Perplex.ContentBlocks#7]
The only way to add/paste content between two existing blocks is to hover the mouse over the space between.

Solutions could include:
- Make the buttons show on hover OR when tab indexed (much like a "skip to content" link)
- Make the buttons always visible
- Make a smaller version of the buttons always visible, which then expands on hover or focus

Does anyone have any preferences here?

**Comment by PerplexDaniel** [comment]
Closed due to 4 weeks of inactivity.
Feel free to reopen with new input.

**Comment by glombek** [comment]
This issue should not be closed due to inactivity - these will take time to fix but remain severe accessibility issues.
